src/error/OperationCancelledError.ts

```diff
@@ -0,0 +1,11 @@
+class OperationCancelledError extends Error {
+  readonly cancelled = true;
+
+  constructor(message: string) {
+    super(message);
+
+    Object.setPrototypeOf(this, OperationCancelledError.prototype);
```

Comment:
It's only because I'm extending base `Error` class - normally I would also avoid inheritence. The explanation of why it's needed is here: https://stackoverflow.com/a/41102306/11506703

Reply:
Interesting! I wonder if we need to transpile to es5 anymore given the minimum supported node version is 8.6? Doesn't that mean es6 is supported anyway?

Reply:
Currently, we support node >=6.11.5. We can upgrade to node 8 as it will be major release and node 6 is no longer supported :)

Reply:
Cool - does that mean we can change our emit to es6 and remove `Object.setPrototypeOf(this, OperationCancelledError.prototype)`? As I understand it it's only required for es5 transpilation reasons?

By the way, as of December node 8 is also no longer supported too! ☺️

https://blog.risingstack.com/update-nodejs-8-end-of-life-no-support/

Reply:
Yes, it seems that we don't need that for node 8 :) I will change the minimum node version